src/CMakeLists.txt

```diff
@@ -69,6 +66,15 @@ if(FLB_STATS)
     )
 endif()
 
+if(FLB_BUFFERING)
```

Comment:
> what's your opinion about the inclusion for buffering headers ?

I think it is proper.
Codes which needs these header is a part of `flb_engine.c` and `flb_task.c`.
So, we have to classify in source code, not in CMakeLists.txt.

On the other hand, `flb_buffer*.c` is completely not needed when `FLB_BUFFERING` is No.